axelrod/tests/strategies/test_axelrod_first.py

```diff
@@ -138,6 +138,31 @@ def test_strategy(self):
         self.versus_test(axelrod.Defector(), expected_actions=actions)
 
 
+class TestGraaskamp(TestPlayer):
+
+    name = "Graaskamp"
+    player = axelrod.Graaskamp
+    expected_classifier = {
+        'memory_depth': 57,
+        'stochastic': True,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        # Play against opponents
+        actions = [(C, C), (C, D), (D, C), (C, D), (D, C)]
```

Comment:
These tests are a good start, once you've made the changes I've requested I think we'll need to change one or two and possibly include a few more to ensure we're checking all the described behaviours :) :+1:

Reply:
I've added some more tests :) I think I've found a bug. After round 57 when playing with Grudger, Graaskamp only cooperates. I'm trying to fix this.